dev/bots/suite_runners/run_framework_coverage_tests.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show File;
+
+import 'package:path/path.dart' as path;
+
+import '../test.dart';
```

Comment:
This circular dependency (test.dart -> run_framework_coverage_tests.dart -> test.dart) is odd. Should the `runFlutterTest` live in a separate file that can be used by both?